drop-boxes/register-nanopore-dropbox/register-nanopore.py

```diff
@@ -80,7 +78,7 @@ def createNewExperiment(transaction, space, project):
     newExpID = None
     while expExists:
         run += 1
-        newExpID = '/' + space + '/' +project+ '/' + project+str(run)
+        newExpID = '/' + space + '/' +project+ '/' + project+'E'+str(run)
```

Comment:
Why add 'E' ?

Reply:
the last part is the experiment code, which is project+the letter "E" + a running number. this was a bug unrelated to the checksum feature